src/FSharp.Data.GraphQL.Shared/Errors.fs

```diff
@@ -187,7 +212,15 @@ type GQLProblemDetails = {
             | :? ICoerceGQLError as error -> error.VariableMessage + error.Message
             | _ -> error.Message
 
-        GQLProblemDetails.Create (message, extensions)
+        match error with
+        | :? IGQLExceptionError as exceptionError ->
+            match extensions with
+            | Include x ->
+                GQLProblemDetails.CreateOfException(exceptionError.Message, exceptionError.Exception, extensions = x)
+            | Skip ->
+                GQLProblemDetails.CreateOfException(exceptionError.Message, exceptionError.Exception)
```

Comment:
The same as below

Reply:
It's not possible. See my previous comment.

Reply:
Done in your PR. Closing comment